Ability to set API Key & API Key Secret manually

EricFrancis12 opened the pull request:
These changes enable setting Twitter API Key & API Key Secret manually, without removing any existing functionality.

michimani commented on client.go:
```diff
@@ -38,6 +38,8 @@ type NewClientInput struct {
 	AuthenticationMethod AuthenticationMethod
 	OAuthToken           string
 	OAuthTokenSecret     string
+	ApiKey               string
+	ApiKeySecret         string
```
@EricFrancis12 
Thank you for the useful additions!
Your implementation is perfect, including the test code, but I would like to make one correction.

In this library, the keyword “API” is written as `api` or `API` in the code. Therefore, I would like to ask you to also refer to the newly added `ApiKey` and `ApiKeySecret` as `APIKey` and `APIKeySecret`.

michimani replied:
I have confirmed that it has been corrected.
Thank you for your response.

EricFrancis12 commented:
@michimani Thanks for your review.

I corrected the capitalization.

Also, it looks like `SetAPIKey()` and `SetAPIKeySecret()` are not needed after all, as `authorize()` is only called inside of `NewClient()`. So I removed them from the PR.

michimani commented:
@EricFrancis12  

Thank you for your pull request for additional features!

I will merge this PR and incorporate it into the next version.

Once again, thank you for your contribution!
